### DISCUSSION TITLE
Dont ever click "lease"

### DISCUSSION THREAD

**Discussion by seagulmines** [Ride-The-Lightning/RTL#1221] [Troubleshooting Questions] [answered]
Hello folks,
i have all my stuff in line, my seed my password. Unfortunally I have been clicking at "onchain balances", "actions" on "lease" instead of "label". I still have the transaction record and thankfully the adress. How I get back my balance? *sorry I am a noob.
Thankyou for answear!

**Comment by seagulmines** [comment]
Nevermind, my node`s acess on blockchain found it by itself. I am a lucky one. ^^ Thankyou anyway :)

**Reply by seagulmines** [reply]
Mr. Admin, you can delete my question. :)

**Reply by saubyk** [reply, marked as answer]
Hi @seagulmines if you click on `Lease` by accident, the UTXO gets locked for a period of 10 mins only, and will appear again on your wallet balance after 10 mins.

**Reply by seagulmines** [reply]
Thank you! 👍🏻 :D
